connect/src/com/telenor/connect/headerenrichment/HeTokenResponse.java

```diff
@@ -0,0 +1,27 @@
+package com.telenor.connect.headerenrichment;
+
+import java.util.Date;
+
+public class HeTokenResponse {
```

Comment:
Have you considered using Lombok for some of the boiler plate code? E.g. for both constructor and getters and setters in this class.

Reply:
Not done a full assessment, no. I'd be willing to consider it, but would in that case like to the change in a separate pull request.

Reply:
Good idea to keep it a separate commit. If at all important enough to change...